### DISCUSSION TITLE
Can you use Upscayl with a virtual gpu?

### DISCUSSION THREAD

**Discussion by AjoStarry** [upscayl/upscayl#845] [Q&A]
Hello,

I see that there is a workaround for using Upscayl on a Windows PC with no gpu, but I was wondering if it was possible to use Upscayl with a Virtual GPU? (Like the ones for rent on Cudo Compute?)

**Comment by aaronliu0130** [comment]
If it behaves exactly as a graphical GPU, you should be fine. Why not try it?
